app/uk/gov/hmrc/apigatekeeperapisfrontend/views/ApiListPage.scala.html

```diff
@@ -16,12 +16,14 @@
 
 @import uk.gov.hmrc.apiplatform.modules.apis.domain.models._
 @import uk.gov.hmrc.apiplatform.modules.gkauth.domain.models.LoggedInRequest
+@import uk.gov.hmrc.apigatekeeperapisfrontend.controllers.routes
 
 @this(layout: Layout, govukTable: GovukTable)
 
 @(apis: List[ApiDefinition])(implicit request: LoggedInRequest[_])
 
 @layout(pageTitle = Some("APIs")) {
+    <a href="@routes.ApiListController.csv" class="govuk-button" download>Download CSV</a>
```

Comment:
Does this mean the file name will be `Download CSV.csv`?  Could/should we include the date in the file name (would it be helpful)?  I realise this was not in the requirements but it would reduce name clashes when downloading resulting in names like `Download CSV (1).csv`.  Ignore if this is overkill.